probe-rs/src/architecture/arm/sequences.rs

```diff
@@ -597,7 +597,7 @@ pub trait ArmDebugSequence: Send + Sync + Debug {
 
             let start = Instant::now();
             let mut timeout = true;
-            while start.elapsed() < Duration::from_micros(100_0000) {
+            while start.elapsed() < Duration::from_micros(1_000_000) {
```

Comment:
I think `Duration::from_secs(1)`would be clearer here.

Reply:
Thanks, I went ahead and touched up the rest, too.